build.sh

```diff
@@ -15,6 +15,22 @@ unset CALLBACK
 
 # Run build process based on configuration files
 
+# load nvm
+. "$NVM_DIR/nvm.sh"
+
+# use .nvmrc if it exists
+if [[ -f .nvmrc ]]; then
+  nvm install >&   /dev/null
```

Comment:
I'm not quite sure how to do the bash redirection here to only get the `stderr` and none of the `stdout`